frontend/lib/src/components/elements/Video/Video.test.tsx

```diff
@@ -41,75 +43,71 @@ describe("Video Element", () => {
+    render(<Video {...props} />)
+    const video = screen.getByTestId("stVideo")
 
-    expect(videoWrapper.prop("className")).toContain("stVideo")
-    expect(videoWrapper.prop("style")).toStrictEqual({
-      width: props.width,
-      height: 528,
-    })
+    expect(video).toHaveAttribute("class", "stVideo")
+    expect(video).toHaveStyle("width: 0px; height: 528px;")
   })
 
   it("has controls", () => {
     const props = getProps()
-    const wrapper = mount(<Video {...props} />)
+    render(<Video {...props} />)
 
-    expect(wrapper.find("video").prop("controls")).toBeDefined()
+    expect(screen.getByTestId("stVideo")).toHaveAttribute("controls")
   })
 
   it("creates its `src` attribute using buildMediaURL", () => {
-    const wrapper = shallow(
-      <Video {...getProps({ url: "/media/mockVideoFile.mp4" })} />
-    )
-    const videoElement = wrapper.find("video")
+    render(<Video {...getProps({ url: "/media/mockVideoFile.mp4" })} />)
     expect(buildMediaURL).toHaveBeenCalledWith("/media/mockVideoFile.mp4")
-    expect(videoElement.prop("src")).toBe("https://mock.media.url")
+    expect(screen.getByTestId("stVideo")).toHaveAttribute(
+      "src",
+      "https://mock.media.url"
+    )
   })
 
   describe("YouTube", () => {
     it("renders a youtube iframe", () => {
       const props = getProps({
         type: VideoProto.Type.YOUTUBE_IFRAME,
       })
-      const wrapper = mount(<Video {...props} />)
-      const iframeWrapper = wrapper.find("iframe")
-
-      expect(iframeWrapper.props()).toMatchSnapshot()
+      render(<Video {...props} />)
+      expect(document.body).toMatchSnapshot()
     })
 
     it("renders a youtube iframe with an starting time", () => {
       const props = getProps({
         type: VideoProto.Type.YOUTUBE_IFRAME,
         startTime: 10,
       })
-      const wrapper = mount(<Video {...props} />)
-      const iframeWrapper = wrapper.find("iframe")
-
-      expect(iframeWrapper.props()).toMatchSnapshot()
+      render(<Video {...props} />)
+      expect(document.body).toMatchSnapshot()
     })
   })
 
   describe("updateTime", () => {
     const props = getProps()
-    const wrapper = mount(<Video {...props} />)
-    const videoElement: HTMLVideoElement = wrapper.find("video").getDOMNode()
 
-    it("sets the current time to startTime on mount", () => {
+    it("sets the current time to startTime on render", () => {
+      render(<Video {...props} />)
+      const videoElement = screen.getByTestId("stVideo") as HTMLMediaElement
       videoElement.dispatchEvent(new Event("loadedmetadata"))
       expect(videoElement.currentTime).toBe(0)
     })
 
     it("updates the current time when startTime is changed", () => {
-      wrapper.setProps(getProps({ startTime: 10 }))
+      const { rerender } = render(<Video {...props} />)
+      const videoElement = screen.getByTestId("stVideo") as HTMLMediaElement
+
       videoElement.dispatchEvent(new Event("loadedmetadata"))
```

Comment:
Same, can be removed.
Would also add check before rerender:
```javascript
expect(videoElement.currentTime).toBe(0)

rerender(<Video {...getProps({ startTime: 10 })} />)
expect(videoElement.currentTime).toBe(10)
```

Reply:
Wouldn't the test above check for the same exact thing? I would argue probably not necessary.

Reply:
It does in isolation, adding it is just clearer confirmation that before and after states render as expected